src/test/java/core/basesyntax/service/impl/FileServiceImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.FileService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class FileServiceImplTest {
+    private static final String CORRECT_FILE_PATH = "src/test/java/resources/testInput.csv";
+    private static final String NOT_EXISTING_FILE_PATH = "src/test/java/resources/notExisting.csv";
+    private static final String INCORRECT_FILE_PATH = "src/test/java/reso/testOutput.csv";
```

Comment:
**resources** location is really important for some cases
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/20326858/a0f9763a-b912-45ab-ae85-e4688954681e)